tests/modelbench_tests/test_benchmark_runner.py

```diff
@@ -36,26 +36,35 @@ def fake_all_secrets(value="some-value") -> RawSecrets:
     return raw_secrets
 
 
+class FakeExplodingAnnotator(FakeAnnotator):
```

Comment:
This pollutes a global variable with test-specific data, and as a general rule tests should not have side effects.

Reply:
In general, I prefer to avoid global variables, which neatly solves this problem. But since fixing that is probably out of scope for this, I think the next-best solution is either setup/teardown (which add the fakes and then restore state after the test has run) or mocking such that the test diverts the production code so that it uses a fake registry which is discarded after the test has run.

Reply:
That makes sense! Updated.